### PR TITLE
Design updates

### PR DESCRIPTION
**Pull request by maxf**

### CODE REVIEW

File: request_a_govuk_domain/request/forms.py [hunk 1]
```diff
@@ -149,6 +149,16 @@ def __init__(self, *args, **kwargs):
         self.helper.layout = Layout(
             Fieldset(
                 Field.text("domain_name"),
+                DomainsHTML(
```
Comment (prakash-gds) [inline request_a_govuk_domain/request/forms.py:152]: I think DomainsHTML.warning method should work here?

File: request_a_govuk_domain/request/forms.py [hunk 2]
```diff
@@ -216,9 +229,21 @@ def __init__(self, *args, **kwargs):
             ),
             Fieldset(
                 DomainsHTML('<h2 class="govuk-heading-m">Contact details</h2>'),
+                DomainsHTML(
```
Comment (prakash-gds) [inline request_a_govuk_domain/request/forms.py:232]: Could DomainsHTML.p work here?

File: request_a_govuk_domain/request/forms.py [hunk 3]
```diff
@@ -216,9 +229,21 @@ def __init__(self, *args, **kwargs):
...
+                    '<p class="govuk-body">We are collecting the registrant’s personal contact details to confirm their identity.</h2>'
+                ),
                 Field.text("registrant_full_name", field_width=20),
                 Field.text("registrant_phone", field_width=20),
                 Field.text("registrant_email"),
+                DomainsHTML(
```
Comment (prakash-gds) [inline request_a_govuk_domain/request/forms.py:238]: Please see previous comment about using warning method.

File: request_a_govuk_domain/request/forms.py (hunk 1 above)
Comment (maxf) [reply request_a_govuk_domain/request/forms.py]: Thanks. I'd forgotten about those.

File: request_a_govuk_domain/request/forms.py (hunk 3 above)
Comment (maxf) [reply request_a_govuk_domain/request/forms.py]: Changed.

**Review by prakash-gds — APPROVED** [review]

**Pull request merged**